content/publication-bias.md

```diff
@@ -0,0 +1,27 @@
+# Publication bias
+
+Publication bias is the selective publication of research results on an arbitrary judgement. Such arbitrary judgements (can) vary across individuals, over time, and remain undisclosed to those who are not privy to the decision process. Publication bias distorts the complete picture of research results and by extension threatens the validity of results, theories, and effectivity of interventions. For example, [less than half of all registered clinical trials on breast cancer are published](https://doi.org/10.1186/s41073-016-0017-4) for unknown reasons.
+
+The most well-known form of publication bias is based on statistical significance: Significant results (p < .05) have a higher probability of getting submitted- and accepted for publication than nonsignificant results. Other forms of publication bias can include higher probabilities of publication for results that confirm prior beliefs or research that uses common methods because they're better understood. 
```

Comment:
Is this a good place to also mention the importance of publishing null results? Or is that what you mean when you refer to nonsignificant results.

I think it would be good to include "null results" specifically at some point, somewhere as a lack of publishing null results leads to a lot of duplication of effort.

Reply:
Good point. Null, negative, nonsignificant all are the same in essence, which I should clarify. I'm thinking how to address the issue of publishing all results. Would that be a separate topic? Seems to not fit in in describing it but at the same time it could.

Maybe something like "[Effects of/Mitigating] publication bias" could be a good additional section? That section could be specific to null results and how to share your results regardless of editorial decisions (e.g., preprints etc :))

Reply:
Added some clarification in the text on null, negative, positive